DOCS-#4027, #4211: Add GIF images and chart to Modin README demonstrating speedups (#4232)
modin-project/modin: README.md

cores. Modin works especially well on larger datasets, where pandas becomes painfully slow or runs
[out of memory](https://modin.readthedocs.io/en/latest/getting_started/why_modin/out_of_core.html).

By simply replacing the import statement, Modin offers users effortless speed and scale for their pandas workflows:

<img src="docs/img/Modin_Pandas_Import.gif" style="display: block;margin-left: auto;margin-right: auto;" width="100%"></img>

> In the GIFs below, Modin (left) and pandas (right) *perform the same pandas* operations on a 2GB dataset. The only difference between the two notebook examples is that Modin uses `import modin.pandas as pd` and pandas uses `import pandas as pd`. 

Reviewer: ```suggestion
In the GIFs below, Modin (left) and pandas (right) perform *the same pandas operations* on a 2GB dataset. The only difference between the two notebook examples is that Modin uses `import modin.pandas as pd` and pandas uses `import pandas as pd`. 
```

<table class="tg">
<thead>
  <tr>
    <th class="tg-0lax" style="text-align: center;"><img src="https://github.com/modin-project/modin/blob/master/docs/img/MODIN_ver2_hrz.png?raw=True" height="35px"></th>
    <th class="tg-0lax" style="text-align: center;"><img src="https://pandas.pydata.org/static/img/pandas.svg" height="50px"></img></th>
  </tr>
</thead>
<tbody>
  <tr>
    <td class="tg-0lax"><img src="docs/img/Modin.gif"></img></td>

>     <td class="tg-0lax"><img src="docs/img/Pandas.gif"></img></td>

Reviewer 2: For both the links to `examples/jupyter` and GIFs, is it better to use absolute links to the URL or local refs? My sense is that absolute links are better so that README still work in isolated contexts. If so, let's replace the links, which means the links will break for now until the PR is merged.

  </tr>
</tbody>
</table>

> The chart below summarizes the runtime speedup of replacing Modin with Pandas based on the examples above. The example notebooks can be found [here](examples/jupyter). To learn more about the speedups you could get with Modin and try out some examples on your own, check out our [10 minute quickstart guide](https://modin.readthedocs.io/en/latest/getting_started/quickstart.html) to try out some examples on your own!

Reviewer 2: ```suggestion
The charts below show the speedup you get by replacing pandas with Modin based on the examples above. The example notebooks can be found [here](examples/jupyter). To learn more about the speedups you could get with Modin and try out some examples on your own, check out our [10-minute quickstart guide](https://modin.readthedocs.io/en/latest/getting_started/quickstart.html) to try out some examples on your own!
```

> The chart below summarizes the runtime speedup of replacing Modin with Pandas based on the examples above. The example notebooks can be located [here](examples/jupyter). To learn more about the speedups you could get with Modin, check out our [10 minute quickstart guide](https://modin.readthedocs.io/en/latest/getting_started/quickstart.html) to try out some examples on your own!

Reviewer: ```suggestion
The chart below summarizes the runtime speedup of replacing Modin with Pandas based on the examples above. The example notebooks can be found [here](examples/jupyter). To learn more about the speedups you could get with Modin and try out some examples on your own, check out our [10 minute quickstart guide](https://modin.readthedocs.io/en/latest/getting_started/quickstart.html)!
```

> <img src="docs/img/Modin_Speedup.svg" style="display: block;margin-left: auto;margin-right: auto;" width="100%"></img>

Reviewer 2: change to absolute links?
